Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -62,7 +62,9 @@ void CGFontSetFTFontSize(id uiFont, void* ftFont, float pointSize, float scale)
 }
 
 CGSize CGFontDrawGlyphsToContext(CGContextRef ctx, WORD* glyphs, DWORD length, float x, float y) {
-    return ctx->Backing()->CGFontDrawGlyphsToContext(glyphs, length, x, y);
```

Comment:
In CoreText these methods would return {} when given a bad parameter, do they do the same in CoreGraphics or do they blow up?